### PR TITLE
Update column docs and package versions

### LINKED ISSUE #4156: Column is missing that `type` is a required field
Currently the documentation makes no mention of the fact that `"type": "Column"` is _required_ when adding a column item to a ColumnSet. It shows it in the example, but it doesn't list it in the property list like it does for every thing else.

---
* URL: https://adaptivecards.io/explorer/Column.html

### PR DESCRIPTION
**Pull request by matthidinger**
## Related Issue
Fixes #4156

## Description

- Fix the metadata for Column.
- Updated package versions to latest patch releases

## How Verified

Ran website locally

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/4273)

### CODE REVIEW

File: schemas/src/elements/Column.json [hunk 1]
```diff
@@ -1,73 +1,73 @@
...
-      "default": "top",
-      "version": "1.1"
-    },
-    "width": {
-      "type": "string|number",
-      "description": "`\"auto\"`, `\"stretch\"`, a number representing relative width of the column in the column group, or in version 1.1 and higher, a specific pixel width, like `\"50px\"`."
-    }
-  }
+	"$schema": "https://raw.githubusercontent.com/microsoft/AdaptiveCards/6f39aedce45864ae1067ed44a5551dc973790bb5/source/nodejs/typed-schema/schema/lib/Type.json",
+	"extends": "Element",
```
Comment (RebeccaAnne) [inline schemas/src/elements/Column.json:3]: >"extends": "Element" [](start = 1, length = 20)

If columns are not toggleable, it looks like the only thing that extends ToggleableItem is Element. In that case, can we remove ToggleableItem and just add Id and isVisible to Element?

File: specs/elements/Column.md [hunk 2]
```diff
@@ -4,13 +4,14 @@
...
 | **items** | `Element[]` | No | The card elements to render inside the `Column`. | 1.0 |
 | **backgroundImage** | `BackgroundImage`, `uri` | No | Specifies the background image. Acceptable formats are PNG, JPEG, and GIF | 1.2 |
 | **bleed** | `boolean` | No | Determines whether the column should bleed through its parent's padding. | 1.2 |
-| **fallback** | `Column`, `FallbackOption` | No | Describes what to do when an unknown item is encountered or the requires of this or any children can't be met. | 1.2 |
+| **fallback** | `Element`, `FallbackOption` | No | Describes what to do when an unknown element is encountered or the requires of this or any children can't be met. | 1.2 |
 | **minHeight** | `string` | No | Specifies the minimum height of the column in pixels, like `"80px"`. | 1.2 |
-| **separator** | `boolean` | No | When `true`, draw a separating line between this column and the previous column. | 1.0 |
-| **spacing** | `Spacing` | No | Controls the amount of spacing between this column and the preceding column. | 1.0 |
+| **separator** | `boolean` | No | When `true`, draw a separating line at the top of the element. | 1.0 |
```
Comment (RebeccaAnne) [inline specs/elements/Column.md:13]: >When `true`, draw a separating line at the top of the element [](start = 35, length = 61)

These changes seem not ideal. I think there's a way to have the description in Column.json override the one coming from Element.

File: specs/elements/Column.md [hunk 3]
```diff
@@ -63,19 +66,34 @@ Specifies the background image. Acceptable formats are PNG, JPEG, and GIF
...
+  * `ImageSet`
+  * `Input.ChoiceSet`
+  * `Input.Date`
+  * `Input.Number`
+  * `Input.Text`
+  * `Input.Time`
+  * `Input.Toggle`
+  * `Media`
+  * `RichTextBlock`
+  * `TextBlock`
```
Comment (RebeccaAnne) [inline specs/elements/Column.md:90]: This isn't correct either. The fallback for a column has to be a column.

File: specs/elements/TextBlock.md [hunk 4]
```diff
@@ -103,6 +103,7 @@ Describes what to do when an unknown element is encountered or the requires of t
 * **Required**: No
 * **Allowed values**:
   * `ActionSet`
+  * `Column`
```
Comment (RebeccaAnne) [inline specs/elements/TextBlock.md:106]: >Column [](start = 5, length = 6)

These changes now indicate that a non-column type (like TextBlock) can fallback to a column. That doesn't seem right.

Comment (matthidinger) [reply specs/elements/TextBlock.md:106]: Thanks Becky, I've filed this to track this and plan to close this PR: https://github.com/microsoft/AdaptiveCards/issues/4312

File: specs/elements/Column.md (hunk 2 above)
Comment (matthidinger) [reply specs/elements/Column.md:13]: Ah I agree, this is actually a pretty poor content regression, let me see what else can be done

**Comment by paulcam206** [comment]
closing out old PRs - feel free to revive this PR and resubmit for review

**Pull request closed without merging**